### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Sajjon**
Formatting
Grammar & punctuation
Links

This is the same PR as closed #33 (but in all my excitement I accidentally committed from my `master`, branch, this PR is from a feature branch). I've rebased merged #34 onto my branch before applying the changes.

All changes suggested by @xwu (in #33) has been applied here, except an incorrect comma.

In general, I would really like to recommend using [amazing grammar and language precision tool Grammarly](https://app.grammarly.com/), which I used to find some room for improvement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ import Numerics
...
+Because we intend to make it possible to adopt Swift Numerics modules in the standard library at some future point, Swift Numerics uses the same license and contribution guidelines as the Swift project.
 
 ## Process
 Swift Numerics is a standalone library separate from the core Swift project.
-In practice it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
+In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
 
-Swift Numerics uses github issues to track bugs and features, and pull requests for development.
+Swift Numerics uses GitHub issues to track bugs and features and pull requests for development.
```
Comment (xwu) [inline README.md]:
```suggestion
Swift Numerics uses GitHub issues to track bugs and features, and pull requests for development.
```

Comment (stephentyrone) [reply README.md]: +1

**Comment by stephentyrone** [comment]
Except for xiaodi's nit, this LGTM. Please take that fix and I'll merge.

File: README.md (hunk 1 above)
Comment (Sajjon) [reply README.md]: But _", and"_ is grammatically incorrect... Do you like this, it is both grammatically correct and easy to read:
### Alternative 1 - Ampersand
_"Swift Numerics uses GitHub issues to track bugs & features and pull requests for development."_

### Alternative 2 - Slash:
_"Swift Numerics uses GitHub issues to track bugs/features and pull requests for development."_

### Alternative 3 - "or":
_"Swift Numerics uses GitHub issues to track bugs or features and pull requests for development."_

🤔

Comment (stephentyrone) [reply README.md]: Those are all bad.

*Swift Numerics uses GitHub issues to track bugs and features. We use pull requests for development.*

Comment (Sajjon) [reply README.md]: @stephentyrone great, fixed! Thanks!

Comment (xwu) [reply README.md]: > But ", and" is grammatically incorrect...

FWIW, it is grammatically just fine here, whatever the tools try to tell you.

Comment (Sajjon) [reply README.md]: @xwu It is the case of `compound predicate`, see screenshot:

Screenshot 2019-11-08 at 17 05 25

Comment (xwu) [reply README.md]: There's no second verb. Even in the case of a compound predicate, it's fine to use a comma when it's necessary for other reasons. You can ignore the tool.

**Pull request merged**